### ISSUE TITLE
About overfitting during model training

### ISSUE THREAD

**Issue by coder-zc** [hcnoh/knowledge-tracing-collection-pytorch#5]
Hi, thank you very much for your work! I found that most of the models auc converged to the maximum value in the first 20% epoch during the training process, and the auc began to decline, but the train loss decreased all the time. Does this mean that overfitting occurred in the model training process? I hope you can answer this question.

**Comment by hcnoh** [comment]
Yes. you are right. if the training loss decreases but the validation loss or test loss increases, the model has gotten overfitting for the given dataset.
